consul/docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+version: '2'
+services:
+  consul:
+    image: "consul:0.8.0"
+    ports:
+      - "8500:8500"
+  audit:
+    build: ./audit/
+    command: ["node", "/usr/src/app/index.js"]
+    ports:
+      - "8100:8100"
+    volumes:
+      - ./audit/:/usr/src/app/
+    environment:
+      - SERVICE_NAME=audit
+  linkerd:
+      image: buoyantio/linkerd:0.9.1
+      command: ['/config.yaml']
+      links:
+        - consul
+      volumes:
+        - ./linkerd/linkerd.conf.yaml:/config.yaml
```

Comment:
Most of the other examples in the repo go with the naming convention `linkerd.yml` instead of `linkerd.conf.yaml`. The repo also prefers `.yml` over `.yaml`. 

Lastly, I like having an `audit` folder, but the `linkerd` folder seems a bit much, given there's only 1 file in it. Were there linkerd plugins or something else you were interested in seeing in that folder?